hypha/templates/base.html

```diff
@@ -72,109 +72,113 @@
-            {% endif %}
-            <header class="header header--standard {% if page.header_image %}header--has-bg-image{% endif %} {% block header_modifier %}{% endblock %}" {% if header_image %}style="background-image:url('{{ header_image.url }}')"{% endif %}>
-
-                {% block apply_button %}
-                    {% include "includes/apply_button.html" %}
-                {% endblock %}
-
-                <div class="header__inner wrapper wrapper--large">
-                    <a href="{{ settings.utils.SystemMessagesSettings.site_logo_link|default:"/" }}" aria-label="Home link">
-                        {% if settings.utils.SystemMessagesSettings.site_logo_default %}
-                            {% image settings.utils.SystemMessagesSettings.site_logo_default width-215 as logo_default %}
-                            <img class="header__logo header__logo--desktop" src="{{ logo_default.url }}">
-                            {% if settings.utils.SystemMessagesSettings.site_logo_mobile %}
-                                {% image settings.utils.SystemMessagesSettings.site_logo_mobile width-60 as logo_mobile %}
-                                <img class="header__logo header__logo--mobile" src="{{ logo_mobile.url }}">
-                            {% else %}
-                                <img class="header__logo header__logo--mobile" src="{{ logo_default.url }}">
-                            {% endif %}
-                        {% else %}
-                            <svg class="header__logo header__logo--desktop header__logo--desktop-light"><use xlink:href="#logo-desktop"></use></svg>
-                            <svg class="header__logo header__logo--desktop header__logo--desktop-dark"><use xlink:href="#logo-desktop--dark"></use></svg>
-                            <svg class="header__logo header__logo--mobile"><use xlink:href="#logo-mobile"></use></svg>
-                        {% endif %}
-                    </a>
-
-                    <div class="header__inner header__inner--mobile-buttons">
-                        <button class="button js-search-toggle" aria-haspopup="true">
-                            <svg class="header__icon header__icon--open-search header__icon--open-search-menu-closed icon icon--mobile-menu"><use xlink:href="#magnifying-glass"></use></svg>
-                            <svg class="header__icon header__icon--close-search header__icon--close-search-menu-closed icon icon--mobile-menu"><use xlink:href="#cross"></use></svg>
-                        </button>
-                        <button class="button button--left-space js-mobile-menu-toggle" aria-haspopup="true">
-                            <svg class="icon icon--mobile-menu"><use xlink:href="#mobile-menu-toggle"></use></svg>
-                        </button>
-                    </div>
-
-                    <section class="header__menus header__menus--desktop">
-                        {% cache 3600 navigation__primary current_site %}
```

Comment:
It is only this primary nav that should be replaced. Logo etc. should always be in place.